prebuilt_swift_static_framework/prebuilt_swift_static_framework.bzl

```diff
@@ -70,14 +109,14 @@ def _prebuilt_swift_static_framework_impl(ctx):
 
 _prebuilt_swift_static_framework = rule(
     implementation = _prebuilt_swift_static_framework_impl,
-    attrs = dict(
+    attrs = dicts.add(apple_support.action_required_attrs(), dict(
```

Comment:
This doesn't require `dicts.add(...)`. This could use the `dict()` constructor that takes a dict as its first argument, and accepts kwargs to go with it:

```python
attires = dict(apple_support.action_required_attrs(),
    # attrs go here
```

Reply:
Done.